consolidate_media.py

```diff
@@ -69,13 +70,26 @@ def main():
     # Pass 2: Move files to media directory, adding a subset of the sha string
     # to the name as a suffix if it has a sha conflict and updating CNXML
     for media in media_files:
+        # Add the relative path to all image filenames
+        new_media_relpath = os.path.relpath(
+            media_dir,
+            media.resolve().parent
+        )
+
         if media.name not in sha_conflicts:
+            update_cnxml_references(
```

Comment:
We are now parsing, modifying, and re-serializing a module file once per image. I'd be worried about some sort of performance issue in this vs. parsing every module file only once and updating all the images inside that module. Did you experience any significant slowdown? if not noticeable, then it's probably not worth putting effort into.

Reply:
Oh, this is only run once per sync, not once per book build, so this performance issue would have to be significant to be a real issue. I'll approve, since it most likely isn't

Reply:
Yeah, valid point that this is definitely not performant. I didn't qualitatively notice the delay in passing while testing relative to the overall runtime of sync'ing which tends to feel dominated by `neb get`. But I went ahead and did a quick (N=1) measurement and the runtime (measured with `time`) of `consolidate_media.py` seems to be ~50 seconds in this branch vs ~13 on master. So a ~4x slowdown. While this is being used in a batch context where ~35s may not matter much, that's at least annoying on principle if nothing else. I'll refactor a bit / push up another commit that addresses this.

Reply:
[Done](https://github.com/openstax/ce-git-storage-spike/pull/13/commits/bb66228e9f87f43274af3fb77ba70cd944556b8e) -- This measured ~12s (N=1), so at parity with or slightly better than `master`.

Reply:
Nice! Above and beyond!